whoami/src/desktop_env.rs

```diff
@@ -62,7 +62,7 @@ impl Display for DesktopEnv {
             Self::Aqua => "Aqua",
             Self::Ios => "IOS",
             Self::Android => "Android",
-            Self::WebBrowser => "Web Browser",
+            Self::WebBrowser(a) => return write!(f, "WebBrowser ( {} )", a),
```

Comment:
Can you change this to use inline format args without the extra spaces inside the parentheses?

```suggestion
            Self::WebBrowser(a) => return write!(f, "WebBrowser ({a})"),
```

Reply:
Oh yeah, tank you!